### PR TITLE
Remove account_id from listings data

### PR DESCRIPTION
**Pull request by simpledan95**
- Remove account_id from listings data

DMSMVR-3048

### CODE REVIEW

**Comment by owenallenaz** [comment]
@simpledan95 @bsabney We need to wait for Anthony to fix the prepare demo data before we merge this. He's updating the cloudinary stuff and wants to be able to push that through to master. So hold-off on merging this until @swissmant gives the all clear.

**Comment by swissmant** [comment]
@simpledan95 @bsabney @owenallenaz I've merged the Cloudinary changes up. Feel free to merge this one when you need to.

**Review by bsabney — APPROVED** [review]
Reviewed and tested.  LGTM

**Pull request merged**
